src/network/networkd-json.c

```diff
@@ -2,8 +2,9 @@
 
 #include <linux/nexthop.h>
 
-#include "dhcp-server-internal.h"
+#include "dhcp6-internal.h"
 #include "dhcp6-lease-internal.h"
+#include "dhcp-server-internal.h"
```

Comment:
Please do not shuffle the headers. `dhcp-` is earlier than `dhcp6`.

Reply:
Fixed it.